public/javascripts/Choropleth.js

```diff
@@ -99,26 +99,35 @@ function Choropleth(_, $, turf) {
                 if (rates[i].region_id === feature.properties.region_id) {
                     compRate = Math.round(100.0 * rates[i].rate);
                     milesLeft = Math.round(0.000621371 * (rates[i].total_distance_m - rates[i].completed_distance_m));
+
+                    var advancedMessage = '';
+                    if(feature.properties.region_id == 251 ||
```

Comment:
This should be a list of neighborhood id numbers. Then the if condition can be accordingly updated.